ui/components/app/signature-request/index.scss

```diff
@@ -8,7 +8,7 @@
   flex-direction: column;
   min-width: 0;
   width: 408px;
-  height: max-content;
+  height: 100%;
   background-color: var(--color-background-default);
```

Comment:
Hey @eriknson : I am not 100% sure if this will work well in scenario when signature request message is long and content is > 100%.

Reply:
Maybe we should be using min-height?